app/views/topics/show.html.erb

```diff
@@ -7,19 +7,7 @@
 <div class="row">
   <div class="col-md-8">
     <p class="lead"><%= @topic.description %></p>
```

Comment:
You may want to use the `topic` partial here (once you make the changes suggested above)